src/seat.c

```diff
@@ -143,6 +143,25 @@ configure_libinput(struct wlr_input_device *wlr_input_device)
 		wlr_log(WLR_INFO, "dwt configured");
 		libinput_device_config_dwt_set_enabled(libinput_dev, dc->dwt);
 	}
+
+	if (dc->send_events_mode < 0) {
```

Comment:
No need to do this anymore. I can’t quite remember what the construct was before that made me suggest it.
The next line now covers it.